sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -18,7 +18,7 @@ namespace Azure.Core.Pipeline
     {
         private const string AzureSdkScopeLabel = "az.sdk.scope";
         internal const string OpenTelemetrySchemaAttribute = "az.schema_url";
-        internal const string OpenTelemetrySchemaVersion = "https://opentelemetry.io/schemas/1.17.0";
+        internal const string OpenTelemetrySchemaVersion = "https://opentelemetry.io/schemas/1.23.0";
```

Comment:
Is there a way do provide the version without hard-coding it?  This is a non-blocking comment for this PR, but I'm interested in what our plans are for keeping this up to date.  Will it always be a manual change to this internal shared source file?  We have some maintenance challenges with updating shared source files, so wondering if this could be a problem from that angle.

Reply:
discussed offline: the version describes what's written in the code (attribute names). If we change this version, we probably also change what attributes are populated on activities. I.e. this is not configurable and should remain hardcoded.